.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+# Description:
+
+______________
+
+# Tasks:
+
+- [ ] Describe the problem / feature
+- [ ] Write tests
+- [ ] Write code to solve the problem
+- [ ] Get code review from coworkers / friends
+- [ ] [Squash commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)
+
+I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
I didn't justify adding the CoC in the first place and we don't need to justify adding it to the issue template either.